src/codemodder/codemodder.py

```diff
@@ -120,10 +134,16 @@ def run(original_args) -> int:
         )
         return 1
 
+    configure_logger(argv.verbose)
+
+    log_section("startup")
+    # TODO: fix hardcoded version
```

Comment:
`from codemodder import __VERSION__`

Reply:
I could make this change but it doesn't fix the problem. It's actually hardcoded there as `0.1.0` right now. I need to make the change I discussed yesterday to move to tag-based versioning.

Reply:
ah I see what you mean, but then this todo should be where __version__ is